source/includes/reference/_borderless.md

```diff
@@ -88,26 +154,42 @@ creationTime                      | Date when balance account was opened
 balances[n].reservedAmount.value     | Reserved amount from your balance | Decimal
 balances[n].reservedAmount.currency  | Reserved amount currency code       | Text
 balances[n].bankDetails              | Bank account details assigned to your borderless account. Available for EUR, GBP, USD, AUD.  | Group
 balances[n].bankDetails.id           | Bank account details id | Integer 
-balances[n].bankDetails.currency     | Bank account currency | Text 
-balances[n].bankDetails.bankCode     | Bank account code  | Text 
-balances[n].bankDetails.accountNumber | Bank account number | Text 
-balances[n].bankDetails.swift         | Bank account swift code | Text 
-balances[n].bankDetails.iban          | Bank account iban | Text 
-balances[n].bankDetails.bankName      | Bank name | 
-balances[n].bankDetails.accountHolderName           | Bank account holder name | Text
-balances[n].bankDetails.bankAddress.addressFirstLine           | Bank address street and house | Text
-balances[n].bankDetails.bankAddress.postCode           | Bank address zip code | Text
-balances[n].bankDetails.bankAddress.city           | Bank address city | Text 
-balances[n].bankDetails.bankAddress.country           | Bank address country | Text 
-balances[n].bankDetails.bankAddress.stateCode           | Bank address state code | Text 
+balances[n].bankDetails.currency     | Bank account currency | Text
+balances[n].bankDetails.profileId    | Your profileId  | Integer 
+balances[n].bankDetails.accountHolderName        | Bank account holder name | Text
+balances[n].bankDetails.bankCode                   | Bank codes for these bank details (depends on currency) | Group 
+balances[n].bankDetails.bankCode.swiftCode         | BIC/Swift Code (optional) | Text 
+balances[n].bankDetails.bankCode.sortCode          | Sort Code (optional) | Text 
+balances[n].bankDetails.bankCode.bsb               | Sort Code (optional) | Text 
+balances[n].bankDetails.bankCode.achRoutingNumber  | ACH routing number (optional) | Text 
+balances[n].bankDetails.bankCode.wireRoutingNumber | Wire routing number (optional) | Text 
+balances[n].bankDetails.accountNumber            | Account numbers for these bank details | Group
+balances[n].bankDetails.accountNumber.default    | Account number (optional) | Text
+balances[n].bankDetails.accountNumber.iban       | IBAN (optional) | Text
+balances[n].bankDetails.limits                   | Limits for these bank details | Group
+balances[n].bankDetails.limits.daily             | Daily limits | Group
+balances[n].bankDetails.limits.daily.currency    | Daily limits currency | Text
+balances[n].bankDetails.limits.daily.value       | Daily limit amount (can be null if no limit) | Text
+balances[n].bankDetails.limits.yearly            | Yearly limits | Group
+balances[n].bankDetails.limits.yearly.currency   | Daily limits currency | Text
+balances[n].bankDetails.limits.yearly.value      | Daily limit amount (can be null if no limit) | Text
```

Comment:
All values should be `Decimal`s, shouldn't they?
<img width="210" alt="screen shot 2018-10-22 at 10 27 26" src="https://user-images.githubusercontent.com/829783/47286052-16b52f00-d5e5-11e8-97ad-640ba307daf8.png">